lib/fuelsdk/soap.rb

```diff
@@ -319,11 +319,44 @@ def soap_perform object_type, properties, action
       soap_request :perform, message
     end
 
+    def soap_perform object_type, properties, options = {}
+      message = create_action_message 'Definitions', object_type, properties, 'Start'
+
+      response = soap_request :perform, message
```

Comment:
Minor thing but couldn't you simplify the above to lines to: `response soap_perform object_type, properties, 'Start'`?